apps/dashboard/src/utils/parseGroupName.ts

```diff
@@ -7,9 +7,7 @@ import { BigNumber, utils } from "ethers"
  */
 export default function parseGroupName(groupNameBN: string) {
     try {
-        return utils.parseBytes32String(
-            BigNumber.from(groupNameBN).toHexString()
-        )
+        return ethers.decodeBytes32String(BigInt(groupNameBN).toString())
```

Comment:
```suggestion
       return decodeBytes32String(toBeHex(groupNameBN))
```

Reply:
The way on-chain group names are displayed in the dashboard is not correct, this should solve it.